api/db-cdr/generate-cdr/csv/survey_question_map.csv

```diff
@@ -1,82 +1,82 @@
-id,survey_concept_id,question_concept_id
-1,1586134,1586135
-2,1586134,1586140
-3,1586134,1586150
-4,1586134,1586151
-5,1586134,1586152
-6,1586134,1586153
-7,1586134,1586154
-8,1586134,1586155
-9,1586134,1585845
-10,1586134,1585838
-11,1586134,1585899
-12,1586134,1585940
-13,1586134,1585852
-14,1586134,1585892
-15,1586134,1585889
-16,1586134,1585890
-17,1586134,1585386
-18,1586134,43529119
-19,1586134,1585952
-20,1586134,1585961
-21,1586134,1585375
-22,1586134,1585370
-23,1586134,1585879
-24,1586134,1585886
-25,1586134,1585917
-26,1586134,1585929
-27,1586134,1585933
-28,1585710,1585766
-29,1585710,1585772
-30,1585710,1585778
-31,1585710,1585711
-32,1585710,1585717
-33,1585710,1585723
-34,1585710,1585729
-35,1585710,1585735
-36,1585710,1585741
-37,1585710,1585747
-38,1585710,1585748
-39,1585710,1585754
-40,1585710,1585760
-41,1585710,1585784
-42,1585710,1585789
-43,1585710,1585791
-44,1585710,1585795
-45,1585710,1585796
-46,1585710,1585802
-47,1585710,1585811
-48,1585710,1585803
-49,1585710,1585806
-50,1585710,1585815
-51,1585710,1585820
-52,1585855,1585857
-53,1585855,1585860
-54,1585855,1585864
-55,1585855,1585867
-56,1585855,1585870
-57,1585855,1585873
-58,1585855,1586166
-59,1585855,1586169
-60,1585855,1586174
-61,1585855,1586177
-62,1585855,1586182
-63,1585855,1586185
-64,1585855,1586190
-65,1585855,1586193
-66,1585855,1586198
-67,1585855,1586201
-68,1585855,1586207
-69,1585855,1586213
-70,1585855,1585636
-71,1585855,1585649
-72,1585855,1585650
-73,1585855,1585656
-74,1585855,1585662
-75,1585855,1585668
-76,1585855,1585674
-77,1585855,1585680
-78,1585855,1585686
-79,1585855,1585692
-80,1585855,1585698
-81,1585855,1585704
+id,survey_concept_id,question_concept_id,path,is_main
```

Comment:
Is this the same as the otehr CSV? Why are there two?

Reply:
One is used by the scripts during the bq count generation. and the other for local debugging.

Reply:
They are identical. I thought local data load happens from db-cdr/csv and big query data load happens from db-cdr/generate-cdr/csv/. 
Correct me if i am wrong and one would do the job.